packages-content-model/roosterjs-content-model-core/lib/publicApi/domUtils/getCurrentSegmentTextFormat.ts

```diff
@@ -0,0 +1,29 @@
+import type { ContentModelSegment, ContentModelSegmentFormat } from 'roosterjs-content-model-types';
+
+/**
+ * Get the text format of the current selected segment
```

Comment:
Please update the comment to explain what is the difference between the segment format and the result of this function